packages/cli/test/utils/crucible/kurtosis/test/multi-fork.yml

```diff
@@ -1,14 +1,25 @@
 participants:
   - el_type: geth
     cl_type: lodestar
-    count: 4
+    #vc_type: lodestar
+    #cl_extra_params: ["--builder.seelction=default"]
+  - el_type: geth
+    cl_type: lodestar
+    #vc_type: lodestar
+    #cl_extra_params: ["--builder.seelction=default"]
+  - el_type: geth
+    cl_type: lodestar
+    #vc_type: lodestar
+    #cl_extra_params: ["--builder.seelction=executiononly"]
+  - el_type: geth
+    cl_type: lodestar
+    #vc_type: lodestar
+    #cl_extra_params: ["--builder.seelction=default"]
```

Comment:
Should not we just create a network config yaml file for Kurtosis and then load it up here in TS?

Reply:
Was confused with some other file. Ignore this message.